### PR TITLE
Endpoints to GET all deployments for a given model_id

### PR DESCRIPTION
**Pull request by nee1k**
image

### CODE REVIEW

File: ingester/database.py [hunk 1]
```diff
@@ -405,3 +405,13 @@ def get_result_query(self, query, parameters):
...
         return result
+
+    def get_deployments(self, model_id):
+        query = """
+            MATCH (m:Model {model_id: $model_id})-[:HAS_DEPLOYMENT]->(d:Deployment)
+            RETURN properties(d) as deployment_info
+        """
+        with self.driver.session() as session:
+            result = session.run(query, model_id=model_id)
+            record = result.single()
```
Comment (swsachith) [inline ingester/database.py:416]: There's going to be more than one result.

**Review by swsachith — CHANGES_REQUESTED** [review]

File: ingester/database.py (hunk 1 above)
Comment (nee1k) [reply ingester/database.py:416]: fixed.

image

File: server/server.py [hunk 2]
```diff
@@ -108,6 +108,24 @@ def get(self):
         model_card_dict = mc_reconstructor.get_all_mcs()
         return model_card_dict, 200
 
+# Get deployment information
+@api.route('/deployments')
```
Comment (swsachith) [inline server/server.py:112]: This should be called something like model_deployments or get_model_deployments. There are deployments for both models and experiments (and from devices). We should distinguish that somehow.

**Review by swsachith — CHANGES_REQUESTED** [review]

**Comment by swsachith** [comment]
you can remove the images above as they are out-of-date.

**Pull request merged**